vent/controller/control_module.py

```diff
@@ -178,7 +178,7 @@ def _initialize_set_to_COPY(self):
         with self._lock:
         # Copy of the SET variables for threading.
             self.COPY_SET_PIP       = self.__SET_PIP 
-            self.COPY_SET_PIP_TIME  = self.__SET_PIP_TIME
+            self.COPY_SET_PIP_TIME  = self.__SET_PIP_GAIN
```

Comment:
@sneakers-the-rat whatever we want to call this... gain/pip_time/flow... You seem to have a talent for human-machine interactions :-)

Reply:
We call it "flow" on the GUI, and it ranges from 1-5. Tuning it is fun and just seems to work.